atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleTableNameGetterImpl.java

```diff
@@ -118,12 +124,37 @@ private Set<TableReference> getTableReferencesFromShortTableNamesWithPrefix(
                 .collect(Collectors.toSet());
     }
 
-    private Set<String> getLongTableNamesFromTableNames(
-            ConnectionSupplier connectionSupplier, Set<String> shortTableNames) throws TableMappingNotFoundException {
+    private Set<String> getLongTableNames(ConnectionSupplier connectionSupplier, Set<String> shortTableNames) {
+        Map<String, String> shortToLongTableNames =
+                oracleTableNameUnmapper.getShortToLongTableNamesFromMappingTable(connectionSupplier, shortTableNames);
+        Set<String> unmappedTableNames = getUnmappedTableNames(shortToLongTableNames, shortTableNames);
+
         if (useTableMapping) {
-            return oracleTableNameUnmapper.getLongTableNamesFromMappingTable(connectionSupplier, shortTableNames);
+            logIfUnmappedTablesPresent(unmappedTableNames);
+            return new HashSet<>(shortToLongTableNames.values());
+        }
+
+        return unmappedTableNames;
+    }
+
+    private Set<String> getUnmappedTableNames(
+            Map<String, String> shortToLongTableNameMapping, Set<String> expectedShortTableNames) {
+        Set<String> mappedShortTableNames = shortToLongTableNameMapping.keySet().stream()
+                .map(tableName -> tableName.toLowerCase(Locale.ROOT))
+                .collect(Collectors.toSet());
+        return expectedShortTableNames.stream()
+                .filter(tableName -> !mappedShortTableNames.contains(tableName.toLowerCase(Locale.ROOT)))
```

Comment:
nit: might be worth declaring a static `LOWER_CASING_FUNCTION`, to avoid any cases where people accidentally change one but not the other

Reply:
Just added as a static function